src/main/java/net/syscon/elite/service/PrisonToPrisonMoveSchedulingService.java

```diff
@@ -71,6 +72,7 @@ public PrisonToPrisonMoveSchedulingService(final Clock clock,
     @Transactional
     @VerifyBookingAccess
```

Comment:
This this needs `@VerifyBookingAccess(overrideRoles = "PRISON_MOVE_MAINTAINER")`